README.md

```diff
@@ -1,15 +1,105 @@
 # Validations with `form_for`
 
-## Objectives
+Now that we know Rails automatically performs validations defined on models,
+let's use this information to easily display validation errors to the user.
+
+# Objectives
+
+After this lesson, you'll be able to...
 
 - use form_for to display a form with Validations
 - print out full error messages above the form
 - customize aspects of the text_field error message if possible
 
-## Notes
+# `form_for`
+
+This step will make heavy usage of `form_for`, the high-powered alternative to
+`form_tag`. The biggest difference beteen these two helpers is that `form_for`
+creates a form specifically **for** a model object. `form_for` is full of
+convenient features.
+
+A basic implementation looks like this:
+
+```erb
+<!-- app/views/posts/new.html.erb //-->
+
+<%= form_for @post, url: {action: "create"} do |f| %>
```

Comment:
I break this line down a bit. maybe something about how `@post` is the object this form is for. then what's the `url: {action: "create"}`. and what is `f`?